setup.py

```diff
@@ -63,7 +63,7 @@ def reduce_requirements(key):
         'onnx_chainer.functions',
         'onnx_chainer.testing',
     ],
-    version='1.4.1',
+    version='1.5.0',
```

Comment:
(I don't have string opinion) onnx-chainer (and most Chainer family may) updates version number with git release tag.